uamqp/constants.py

```diff
@@ -186,8 +186,6 @@ class ManagementExecuteOperationResult(Enum):
 
     OK = 0
     ERROR = 1
-    FAILED_BAD_STATUS = 2
-    INSTANCE_CLOSED = 3
```

Comment:
These are no longer used?

Reply:
I was thinking of just letting `ManagementExecuteOperationResult` indicate the result of the mgmt operation itself, e.g. mgmt request succeeds or mgmt request fails and leaving the handling of the status code completely to the upper layer.

But after revisiting the existing uamqp design, I think keeping those status code does no harm, it won't prevent the upper layer handle the raw response messages while it keeps clarity for the response. I will revert the code.